synapse/lib/stormtypes.py

```diff
@@ -7087,7 +7087,7 @@ async def liftByProp(self, propname, propvalu=None, propcmpr='='):
                     yield await self.runt.view._joinStorNode(nid)
                 return
 
-            norm, info = await ptyp.norm(propvalu)
+            norm, info = await ptyp.norm(propvalu, view=False)
```

Comment:
You have a bunch of `view=False` in this file which feels weird. I think `view=None` would make more sense.

Reply:
Makes sense. That was the piece I was missing. Maybe add that as a comment in types.py where you do the `if view is None:`?